src/server/api/routes/gameInstances.router.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const gameInstancesController = require('../controllers/gameInstances.controller');
+
+/**
+ * @swagger
+ * /game-instances:
+ *  get:
+ *    summary: Get all the game instances.
+ *    description:
+ *      Will return all game instances.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
```

Comment:
I still think it's confusing to use .then functions in one place and async/await in another